### PR TITLE
chore(deprecate): update readme with deprecation notes

### PR DESCRIPTION
**Pull request by kmova**
- Update readme with new location of code
- Add deprecate note to all user facing CRDs

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,62 @@
...
 
-*Maya* orchestration and management capabilities are delivered through a set of services and tools. Currently, these services support deploying the CAS Storage Solutions in Kubernetes Clusters. In future, these can be extended to support other Container Orchestrators.
+As the OpenEBS community started to add new engines, the engine specific control plane components have been moved to their respective repositories.
+- cStor operators and CSI driver have moved to [openebs/cstor-operators](https://github.com/openebs/cstor-operators) and [openebs/cstor-csi](https://github.com/openebs/cstor-csi) respectively.
+- Jiva operator and CSI driver have moved to [openebs/jiva-operator](https://github.com/openebs/jiva-operator)
+- Local PV Hostpath and Device provisioner has been moved to [openebs/dynamic-localpv-provisioner](https://github.com/openebs/dynamic-localpv-provisioner)
+- Jiva and cStor prometheus metrics exporter been moved to [openebs/m-exporter](https://github.com/openebs/m-exporter)
+- `mayactl` for displaying status of jiva and cstor volumes is merged into [openebs/openebsctl](https://github.com/openebs/openebsctl)
```
Comment (niladrih) [inline README.md:17]: jiva and cstor --> Jiva and cStor

Comment (niladrih) [reply README.md:17]: Similar changes for the lines below.

**Review by niladrih — APPROVED** [review]
LGTM. Minor comment.

File: README.md (hunk 1 above)
Comment (kmova) [reply README.md]: good catch. Addressed!

**Review by niladrih — APPROVED** [review]
LGTM

**Pull request merged**